CurriculaService/src/main/java/com/revature/repositories/VisualizationDao.java

```diff
@@ -0,0 +1,15 @@
+package com.revature.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.revature.models.Visualization;
+
+
+
+
+
+public interface VisualizationDao  extends JpaRepository<Visualization, Integer> {
+	public Visualization findVisualizationById(int id);
```

Comment:
This function is already provided by default in the JpaRepository, as getOne() or findById()

Reply:
I deleted line 12